roles/sympa-fcgi/templates/service.j2

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description = {{ unit_description }} (service instance %i)
+After = syslog.target
+BindTo = sympa.service
```

Comment:
On upstream link between sympa and web services has been weaken, like
``` code
After = syslog.target sympa.service
```
cf. sympa-community/sympa#525

Reply:
Fixed through 7b05561. Thanks @ikedas !